app/controllers/api/v1/lists_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class Api::V1::ListsController < Api::V1::ApiController
+  RESOURCE = 'lists'
+
+  def index
+    @lists = ClearstreamClient::BaseClient.new(api_key: @api_key,
+                                               resource: RESOURCE).index
+  end
+
+  def create
+    @list = ClearstreamClient::List.new(params)
+    if !@list.valid?
+      error_json = return_error(@list.errors)
+      render json: error_json, status: 422
+    else
+      ClearstreamClient::BaseClient.new(data: @list.to_h,
+                                        resource: RESOURCE).create
+    end
+  end
+
+  def show
+    @list = ClearstreamClient::BaseClient.new(api_key: @api_key,
+                                              resource: RESOURCE).show(params[:id])
+  end
+
+  def destroy
+    @list = ClearstreamClient::BaseClient.new(api_key: @api_key,
```

Comment:
If we start this way, then line up the next line with `api_key`, it looks nicer and still satisfies rubocop...

```suggestion
    @list = ClearstreamClient::BaseClient.new(
      api_key: @api_key,
```